### PR TITLE
Implemented read endpoints for solutions API. Related to #35.

### PR DESCRIPTION
**Pull request by joelcox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,6 +279,8 @@ Not implemented: http://developers.freshdesk.com/api/#sla-policies
 - Davin Smith [@davinthesmith]
 - John Williamson [@velua]
 - Ori Roniger [@roniger]
+- Joël Cox [@joelcox]
+
```
Comment (arjunkomath) [inline README.md]: nitpick: unnecessary new line

File: lib/client.js [hunk 2]
```diff
@@ -670,6 +670,34 @@ class Freshdesk {
 		makeRequest('DELETE', this._auth, `${this.baseUrl}/api/v2/time_entries/${id}`, null, null, cb)
 	}
 
+	listAllSolutionCategories(cb) {
```
Comment (arjunkomath) [inline lib/client.js]: Missing inline documentation for all the methods

**Comment by joelcox** [comment]
Thanks for the review. Will fix these hopefully next week.

**Comment by arjunkomath** [comment]
> Thanks for the review. Will fix these hopefully next week.

👍 let me know if you need any help.

**Comment by arjunkomath** [comment]
Closing stale PR

**Pull request closed without merging**